roles/jenkins-worker/files/etc/containers/registries.conf

```diff
@@ -0,0 +1,23 @@
+# For more information on this configuration file, see containers-registries.conf(5).
+#
+# # An array of host[:port] registries to try when pulling an unqualified image, in order.
+# unqualified-search-registries = ["example.com"]
+unqualified-search-registries = ["mirror.gcr.io"]
+short-name-mode = "enforcing"
+
+# Mirror the registry we are targeting for "unqualified" searches with our Nexus mirror
+[[registry]]
+location = "mirror.gcr.io"
+[[registry.mirror]]
+location = "nexus-load-balancer-c4cf05fd92f43ef8.elb.us-east-1.amazonaws.com:8181"
```

Comment:
We'll probably want to use some internal DNS with a more stable name, because I don't look forward to re-generating images just because this name changed.